pages/global-profile/views/index.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import sortBy from 'lodash/sortBy';
+import { Header, PanelList, Link, ExpandingPanel, Snippet } from '@asl/components';
+
+import dateFormatter from 'date-fns/format';
+import { dateFormat } from '../../../constants';
+
+import Profile from '../../profile/read/views/profile';
+import Modules from '../../profile/read/views/modules';
+import RelatedTasks from '../../task/list/views/related-tasks';
+import AsruRoles from '../components/asru-roles';
+
+const formatDate = (date, format) => (date ? dateFormatter(date, format) : '-');
+
+class Index extends React.Component {
+
+  render () {
+
+    const model = this.props.model;
+    const isOwnProfile = this.props.isOwnProfile;
+    const hasEstablishments = !!(model.establishments || []).length;
+
+    const showEmail = this.props.asruUser || model.asruLicensing || this.props.isOwnProfile;
```

Comment:
Is there a reason we show emails for licensing officers but not inspectors?

(sorry if this is in the ticket)

Reply:
yes - it's in the ticket

Reply:
Wild!